### ISSUE TITLE
Increase the code coverage of unit test.

### ISSUE THREAD

**Issue by jinzhejz** [kubernetes-retired/kube-batch#67]
There are not any unit test cases for this project. Need to add them to each go file.

**Comment by k82cn** [comment]
@jinzhejz , please help to append code coverage of current code; I'd like to have at least ~60% coverage for 0.1 release.

**Comment by jinzhejz** [comment]
The code coverage of current code:

```
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/policy/drf coverage: 94.3% of statements

Coverage details:
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/policy/drf/drf.go         (98.5%)
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/policy/drf/podset_info.go (87.8%)
```

```
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/cache      coverage: 30.6% of statements

Coverage details:
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/cache/cache.go         (14.1%)
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/cache/node_info.go     (64.5%)
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/cache/pdb_info.go      (100.0%)
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/cache/pod_set.go       (64.7%)
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/cache/queue_info.go    (63.3%)
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/cache/resource_info.go (50.0%)
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/cache/utils.go         (80.0%)
```

There is no ut cases for following code now:

```
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/apis
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/client
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/controller
github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/policy/drf/util
```
It is better to add some ut cases for `github.com/kubernetes-incubator/kube-arbitrator/pkg/batchd/policy/drf/util` later.

For `apis`,`client`,`controller`, they are used to define internal/client apis and control flow, so I think their ut cases are low priority now.

**Comment by etiennecoutaud** [comment]
WDYT of adding code coverage metrics to CI pipeline to follow this indicator? something like codecov.io ? We could add the label in the README.

In parallel, we could add a Makefile flags (`make test-coverage`) to improve dev experience and have coverage feedback locally.

**Comment by k82cn** [comment]
@etiennecoutaud , that's great :)

**Comment by k82cn** [comment]
Moved to 0.3 because of effort.
